hail/python/hailtop/utils/utils.py

```diff
@@ -108,11 +108,13 @@ def digits_needed(i: int):
 
 
 def grouped(n, ls):
+    if n < 1:
+        raise ValueError ('invalid value for n')
     while len(ls) != 0:
         group = ls[:n]
         ls = ls[n:]
         yield group
-
+        
```

Comment:
Can you get rid of this extra spacing here? This change should not show up in the diff. Try putting your cursor to the end of the line and then hitting backspace to get rid of the extra spaces.

Reply:
Fixed